java/create-terrain-from-a-local-raster/README.md

```diff
@@ -0,0 +1,57 @@
+# Create terrain from a local raster
+
+Set the terrain surface with elevation described by a raster file.
+
+![Create terrain from a local raster](create-terrain-from-a-local-raster.gif)
+
+## Use case
+
+The terrain surface is what the basemap, operational layers, and graphics are draped on. Supported raster formats include:
+
+* ASRP/USRP
+* CIB1, 5, 10
+* DTED0, 1, 2
+* GeoTIFF
+* HFA
+* HRE
+* IMG
+* JPEG
+* JPEG 2000
+* NITF
+* PNG
+* RPF
+* SRTM1, 2
```

Comment:
This might be a stupid question, but should .tif and .tiff be included in this list?

Reply:
I'm not sure @TADraeseke , I just copied what was in the design [README](https://devtopia.esri.com/runtime/common-samples/tree/master/designs/CreateTerrainSurfaceFromALocalRaster).